### PR TITLE
Update docs and sample with guidance for managing multiple accounts

### PR DESCRIPTION
**Pull request by tnorling**
- Updates docs with guidance for managing multiple accounts using the active account APIs
- Also adds an account picker to the react-router-sample demonstrating one way how these APIs can be used to switch between accounts

In response to #3189

### CODE REVIEW

File: lib/msal-angular/docs/FAQ.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ We recommend setting the active account:
...
 
-**Note:** Prior to `@azure/msal-browser@2.15.0` active account did not persist across page loads. If you are using `@azure/msal-browser@2.14.2` or earlier we recommend that you set the active account for each page load. In version 2.15.0 and above the active account will be cached in the cache location specified in your MSAL config and retrieved each time `getActiveAccount` is called.
+**Note:** Prior to `@azure/msal-browser@2.16.0` active account did not persist across page loads. If you are using `@azure/msal-browser@2.15.0` or earlier we recommend that you set the active account for each page load. In version 2.15.0 and above the active account will be cached in the cache location specified in your MSAL config and retrieved each time `getActiveAccount` is called.
```
Comment (derisen) [inline lib/msal-angular/docs/FAQ.md:100]:
```suggestion
**Note:** Prior to `@azure/msal-browser@2.16.0` active account did not persist across page loads. If you are using `@azure/msal-browser@2.15.0` or earlier we recommend that you set the active account for each page load. In version 2.16.0 and above the active account will be cached in the cache location specified in your MSAL config and retrieved each time `getActiveAccount` is called.
```

**Review by derisen — APPROVED** [review]
Much needed, thanks! one comment

**Review by pkanher617 — APPROVED** [review]

File: lib/msal-browser/FAQ.md [hunk 2]
```diff
@@ -216,6 +217,13 @@ No, accounts are not sorted nor are they guaranteed to maintain any particular o
 
 No, the account APIs reflect local account state only. If you need to ensure the user has an active session on the server you should call `acquireTokenSilent` or `ssoSilent` and fallback to interaction if needed.
 
+### How can I switch between multiple logged in users?
+
+Deciding which account to use to acquire tokens is app dependent, however, `@azure/msal-browser` provides 2 convenient APIs to help you keep track of which account is currently "active" and should be used for token requests. Once you've determined which account you wish to use you can call the `setActiveAccount()` API to make sure MSAL uses this account for all subsequent requests. If you would like to switch to a different account, simply call `setActiveAccount()` again and pass it the new account you would like to use. If you need to know which account is currently "active" you can use the `getActiveAccount()` API.
+
+You can read more about the account APIs [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/accounts.md)
```
Comment (jo-arroyo) [inline lib/msal-browser/FAQ.md:224]:
```suggestion
You can read more about the account APIs [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/accounts.md).
```

**Review by jo-arroyo — APPROVED** [review]
Looks great! Thanks for the updates. One tiny nit.

**Pull request merged**